### PR TITLE
#167597713 Fix Responsiveness Issue On Mobile Devices

### PR DESCRIPTION
**Pull request by BrianSerem**
## Description ##
Consider the following cases, this PR fixes this. 
# Expected #
The component is responsive and fits well to the screen. Micro-components are to scale and no distortion of any kind is experienced.

# Actual #
The component is not fitting well on devices with smaller screens. This is due to CSS styling mismatch.
## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [ ] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [ ] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#167597713](https://www.pivotaltracker.com/story/show/167597713)

### CODE REVIEW

File: src/app/pages/login/login.component.scss [hunk 1]
```diff
@@ -53,19 +53,28 @@ img.bg {
...
 
 @media only screen and (max-width: 480px) {
+  .big-container{
+    display: flex;
+    width: 104%;
+    height: 32em;
+
+  }
   .container {
+    // display: flex;
```
Comment (Genza999) [inline src/app/pages/login/login.component.scss]: Hey @BrianSerem , nice work, kindly remove out this commented out code

File: src/app/pages/login/login-form/login-form.component.scss [hunk 2]
```diff
@@ -1,59 +1,73 @@
...
+  width: 19rem;
+  font-size: 20px;
+}
+
+.email:focus, .password:focus {
+  outline: none;
+}
+
+.input {
+  /* border: 1px solid #f00;; */
```
Comment (Genza999) [inline src/app/pages/login/login-form/login-form.component.scss]: Kindly remove this commented out code too.

File: src/app/pages/login/login.component.scss (hunk 1 above)
Comment (BrianSerem) [reply src/app/pages/login/login.component.scss]: I have removed the pieces of commented code

File: src/app/pages/login/login-form/login-form.component.scss (hunk 2 above)
Comment (BrianSerem) [reply src/app/pages/login/login-form/login-form.component.scss]: I have worked on this too

**Review by Genza999 — APPROVED** [review]
LGTM

**Review by akram256 — APPROVED** [review]
LGTM

**Comment by kelvinndmo** [comment]
on this fix, you can also work to fix  the config file where you have the localhost url as the baseurl

**Review by kelvinndmo — APPROVED** [review]

**Review by kelvinndmo — APPROVED** [review]

**Review by Victoradukwu — APPROVED** [review]
LFTM

**Review by JamesMudidi — CHANGES_REQUESTED** [review]
@BrianSerem, nice work here. The code LGTM but I tried creating a review app on Heroku and it fails. I wanted to check out the responsiveness of the application. Here is the error I get
Screenshot 2019-07-31 at 7 12 08 PM

**Comment by BrianSerem** [comment]
> @BrianSerem, nice work here. The code LGTM but I tried creating a review app on Heroku and it fails. I wanted to check out the responsiveness of the application. Here is the error I get
> Screenshot 2019-07-31 at 7 12 08 PM

just a minor issue on Heroku, no technicalities with the code. This has since been fixed and you may now knock yourself out.

**Review by CryceTruly — CHANGES_REQUESTED** [review]
Good work @BrianSerem, However on iPhone 6 & up, white space is not consistent on the right and left.

Screenshot 2019-08-01 at 14 09 18

**Pull request merged**